viadot/config.py

```diff
@@ -64,7 +64,8 @@ def get_source_config(key, config=CONFIG):
     if source_configs is not None:
         for source_config in source_configs:
             if key in source_config.keys():
-                return source_configs[source_configs.index(source_config)][key]
+                # return source_configs[source_configs.index(source_config)][key] OBS!!!!!!!!!!!!!
```

Comment:
Please add this to `CHANGELOG.md` with issue number

Reply:
Also add/fix a test (in `test_config.py`)